app/presenters/notifications/view-notification.presenter.js

```diff
@@ -15,7 +15,7 @@ const { formatLongDate, sentenceCase } = require('../base.presenter.js')
  * @returns {object} The data formatted for the view template
  */
 function go(notificationData) {
-  const { messageType, plaintext, personalisation, sendAfter } = notificationData.notification
+  const { messageType, plaintext, personalisation, sendAfter, createdAt } = notificationData.notification
```

Comment:
Destructured functions should be listed alphabetically
```suggestion
  const { createdAt, messageType, personalisation, plaintext, sendAfter } = notificationData.notification
```